config/manifests/bases/kustomization-base.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ocs-osd-deployer.clusterserviceversion.yaml
+commonAnnotations:
+  olm.skipRange: '>=0.0.1 <@VERSION@'
+patches:
+- patch: '[{"op": "replace", "path": "/spec/replaces", "value": "@REPLACES@"}]'
+  target:
+    kind: ClusterServiceVersion
+    name: ocs-osd-deployer.v0.0.0
```

Comment:
Why do we need this `-base` file if we can already work on the actual file?

Reply:
removed